assets/src/scripts/components/hydrograph/axes.js

```diff
@@ -20,13 +21,28 @@ const { currentVariableSelector, yLabelSelector } = require('./timeseries');
  * @return {Object}             {xAxis, yAxis} - D3 Axis
  */
 export const createAxes = function({xScale, yScale}, yTickSize, parmCd) {
+    const formatter = function(d) {
+        if(d.getHours() === 12) {
+            return dateFormatter(d);
+        } else {
+            return null;
+        }
+    };
+
     // Create x-axis
     const xAxis = axisBottom()
         .scale(xScale)
         .ticks(timeDay)
-        .tickFormat(timeFormat('%b %d'))
+        .tickFormat('')
         .tickSizeOuter(0);
 
+    // Create a second x-axis to hold date/time labels
```

Comment:
It would be good to have a comment on the xAxis as to why it's used (I assume it's to draw the tick marks)